Modules/Invoice/Resources/views/tax-report.blade.php

```diff
@@ -25,37 +25,64 @@
                     <th></th>
                     <th>Project</th>
                     <th>Amount</th>
-                    <th>GST</th>
-                    <th>Amount (+GST)</th>
                     <th>Received amount</th>
-                    <th>TDS</th>
+                    @if (request()->input('region') == config('invoice.indian'))
```

Comment:
Update the config as per the changes done.
```suggestion
                    @if (request()->input('region') == config('region.invoice.indian'))
```

Reply:
Changed every where